artpaint/paintwindow/ImageView.cpp

```diff
@@ -367,6 +367,11 @@ void ImageView::MessageReceived(BMessage *message)
 			}
 		}	break;
 
+		case HS_LAYER_TRANSPARENCY_CHANGED:
```

Comment:
The other cases do have curly brackets...

Reply:
I left this one alone for now; about half of the case HAVE curly braces and half don't.  That file needs a cleanup overall.